messages/env.api.md

```diff
@@ -0,0 +1,41 @@
+# summary
+
+Makes an authenticated HTTP request to the Salesforce REST API and prints the response.
```

Comment:
Might be nice to link to the API reference guide: https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/resources_list.htm